test/unit/app/utils/rewards/test_processor.py

```diff
@@ -0,0 +1,23 @@
+from app.server.utils.rewards.processor import get_user_by_id
+from app.server.utils.rewards.processor import get_transfer_account_token
+from app.server.utils.rewards.processor import get_admin_account_to_disburse
+
+
+def test_get_user_by_id(test_client, init_database, create_transfer_account_user):
```

Comment:
Here you are really only testing the sqlalchemy query filter. I think it is not necessary.